workflow-templates/im-deploy-iis-website.yml

```diff
@@ -417,62 +416,32 @@ jobs:
-          github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
-
-      # https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
-      - name: Update Deployment Board
-        if: always()
-        uses: im-open/update-deployment-board@v1.5
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
-          environment: ${{ env.ENVIRONMENT }}
-          board-number: '' # TODO: Add the automated deployment board number or remove if not using an automated deployment project board.
-          ref: ${{ env.RELEASE_TAG }}
-          ref-type: 'tag'
-          #deployable-type: '' # TODO:  If there are multiple deployables in the repository, add a string type like MFE/DB/API
-          deploy-status: ${{ steps.conclusion.outputs.workflow_conclusion }}
-          timezone: ${{ env.TIMEZONE }}
-
-      - name: Send Status to Teams
-        if: always()
-        uses: im-open/post-status-to-teams-action@v1.3
-        with:
-          title: ${{ needs.set-vars.outputs.WEBSITE_NAME }} Deployment to ${{ env.ENVIRONMENT }} # TODO Verify this title gives enough info since it will be posted to the team channel
-          workflow-status: ${{ steps.conclusion.outputs.workflow_conclusion }}
-          workflow-type: Deploy
-          teams-uri: ${{ secrets.MS_TEAMS_URI }} # This is a repo-level secret (unless 'environment:' has been added to the job)
-          timezone: ${{ env.TIMEZONE }}
-          # TODO: Verify the custom facts you want included
-          custom-facts: |
-            [
-              { "name": "Workflow", "value": "${{ github.workflow }}" },
-              { "name": "Run", "value": "${{ github.run_id }}" },
-              { "name": "Actor", "value": "${{ github.actor }}" }
-            ]
-
-      - name: Send Status to Deployment Notifications Channel for Prod Deploys
-        if: always() && env.ENVIRONMENT == 'prod' && steps.conclusion.outputs.workflow_conclusion == 'success'
-        uses: im-open/post-status-to-teams-action@v1.3
-        with:
-          title: ${{ needs.set-vars.outputs.WEBSITE_NAME }} Deployment to ${{ env.ENVIRONMENT }} # TODO Verify this title gives enough info since it will be posted to the Prod Notifications channel
-          workflow-status: ${{ steps.conclusion.outputs.workflow_conclusion }}
-          workflow-type: Deploy
-          teams-uri: ${{ secrets.DEPLOY_NOTIFICATIONS_CHANNEL }} # This is an org-level secret
-          timezone: ${{ env.TIMEZONE }}
-          include-default-facts: false # This cuts down on the message size for the prod room
-          custom-facts: |
-            [
-              { "name": "Actor", "value": "${{ github.actor }}" },
-              { "name": "Version", "value": "${{ env.RELEASE_TAG }}" }
-            ]
+    uses: im-practices/.github/.github/workflows/im-reusable-finish-deployment-workflow.yml@v2
```

Comment:
Why the double `.github`?

Reply:
It's in the .github repository and inside of the .github folder in that repo.

Reply:
Ah!

Reply:
It definitely looks typo-y.